### DISCUSSION TITLE
obsidian 不可回避的缺点及其解决方案

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#32] [备发布站的评论系统]
# obsidian 不可回避的缺点及其解决方案

  可以访问此文的双链版本，获得完整阅读体验：Obsidian 不可回避的缺点及其解决方案 — 🌱 oldwinterの数字花园

https://notes.oldwinter.top/obsidian-%E4%B8%8D%E5%8F%AF%E5%9B%9E%E9%81%BF%E7%9A%84%E7%BC%BA%E7%82%B9%E5%8F%8A%E5%85%B6%E8%A7%A3%E5%86%B3%E6%96%B9%E6%A1%88

**Comment by Ravenclawer** [comment]
这个数字花园不错，我之前都是用gridea，没有双链
